benchmarks/cugraph/standalone/README.bulk_sampling.md

```diff
@@ -0,0 +1,116 @@
+# cuGraph Bulk Sampling
```

Comment:
Recommend moving this to a different folder like `bulk_sampling/` for readme to be rendered.